packages/json-schema/lib/main.tsp

```diff
@@ -0,0 +1,115 @@
+import "../dist/src/index.js";
+
+namespace JsonSchema;
+
+/**
+ * Add to namespaces to emit models within that namespace to JSON schema.
+ * Add to another declaration to emit that declaration to JSON schema.
+ *
+ * Optionally, for namespaces, you can provide a baseUri, and for other declarations,
+ * you can provide the id.
+ *
+ * @param baseUri Schema IDs are interpreted as relative to this URI.
+ */
+extern dec JsonSchema(target: unknown, baseUri?: string);
```

Comment:
```suggestion
extern dec jsonSchema(target: unknown, baseUri?: string);
```
no?

Reply:
Somehow the capital makes me happy here but... yes.